### ISSUE TITLE
Rho function not giving exact value in case of Water

### ISSUE THREAD

**Issue by ard3051997** [yoelcortes/biosteam#10]
The value is having an error of about 30%

**Comment by yoelcortes** [comment]
Yes, I know exactly what you're talking about. This is a problem with the [thermo](https://thermo.readthedocs.io/en/latest/) package that biosteam is dependent on. Initially it works well, but due to some cache that goes on, the density gets stuck on a high value (corresponding to a high temperature that the biorefineries run).

This is a difficult problem to deal with due to how _thermo_ is built, so please just ignore it for now. Eventually, we will be phasing out _thermo_ for [_thermosteam_](https://github.com/yoelcortes/thermosteam), which is currently in development. You can expect this update for version in April or May.

In the meanwhile, there is a way to reset the cache in _thermo_:
```python
>>> from biosteam import Chemical
>>> water = Chemical('Water', T=298.15, P=101325)
>>> water.VolumeLiquid.method_P = None
>>> water.VolumeLiquid.TP_cached = (0, 0)
>>> water.rho
997.0483008954374
```
Hope this helps!

**Comment by ard3051997** [comment]
Thank You.

**Comment by yoelcortes** [comment]
Thanks for submitting the bug! I just fixed it in "biosteam==1.0.7". I am closing this issue...
